service_storagefacility/src/test/scala/no/uio/musit/microservice/storagefacility/testhelpers/NodeGenerators.scala

```diff
@@ -173,4 +173,39 @@ trait NodeTypeInitializers {
     )
   }
 
+  def createRoomWithDifferentArea(
+    areaTo: Double,
+    perimeter: Boolean = false,
+    theftProtection: Boolean = false,
+    fireProtection: Boolean = false,
+    waterDamage: Boolean = false,
+    routinesAndContingencyPlan: Boolean = false
+  ): Room = {
```

Comment:
FIXME
To save some typing, you could re-use the `createRoom` function already defined in the file, call `.copy` and set the values you want to be different.

```scala
def createRoomWithDifferentArea(
    areaTo: Double,
    perimeter: Boolean = false,
    theftProtection: Boolean = false,
    fireProtection: Boolean = false,
    waterDamage: Boolean = false,
    routinesAndContingencyPlan: Boolean = false
  ): Room = {
    createRoom().copy(
      name = "MyPrivateRoom",
      areaTo = Some(areaTo),
      securityAssessment = SecurityAssessment(
        perimeter = Some(perimeter),
        theftProtection = Some(theftProtection),
        fireProtection = Some(fireProtection),
        waterDamage = Some(waterDamage),
        routinesAndContingencyPlan = Some(routinesAndContingencyPlan)
      )
    )
  }
```